microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -16,71 +16,6 @@ This is the REST endpoint reference for the Microsoft Edge Add-ons API.  This AP
 
```

Comment:
In section "Check the status of a package upload", cross-link back to the Using article's equivalent section.
Repeat for other sections.

Reply:
All 4 APis are already referenced, Can you please clarify what is the ask here?

Reply:
I plan to add hyperlinks from h2 sections in the Reference article linking to the equivalent sections in the Using article, if they don't exist yet.  I plan to push a commit into this PR.

Reply:
Done, in the sibling PR instead.  Cross-linked as See Also h3 sections.  That is, restored my original design from v1 of these articles.